test/addons-napi/test_general/testInstanceOf.js

```diff
@@ -62,7 +62,7 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
 
   function compareToNative(theObject, theConstructor) {
     assert.strictEqual(addon.doInstanceOf(theObject, theConstructor),
```

Comment:
I'll do it here and anywhere else someone flags in this PR, but not sure about everywhere because this sort of thing is all over our code base already and the churn would be extreme.